plugins/basic/info.yml

```diff
@@ -1,22 +1,22 @@
-# Copyright (c) 2017 Shotgun Software Inc.
+# Copyright (c) 2017 Shotgrid Software Inc.
 #
 # CONFIDENTIAL AND PROPRIETARY
 #
-# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# This work is provided "AS IS" and subject to the Shotgrid Pipeline Toolkit
 # Source Code License included in this distribution package. See LICENSE.
 # By accessing, using, copying or modifying this work you indicate your
-# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
-# not expressly granted therein are reserved by Shotgun Software Inc.
+# agreement to the Shotgrid Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgrid Software Inc.
 
 
 base_configuration:
     # The default configuration that the plugin should use.
     # For documentation and details, see
-    # http://developer.shotgunsoftware.com/tk-core/bootstrap.html#sgtk.bootstrap.ToolkitManager.base_configuration
+    # http://developer.shotgridsoftware.com/tk-core/bootstrap.html#sgtk.bootstrap.ToolkitManager.base_configuration
```

Comment:
This leads to an unsecure link. Please replace http with https
Can you make a pass everywhere please? (including `com.unity.integrations.shotgrid`)